virtual_rainforest/models/animals/animal_model.py

```diff
@@ -127,5 +131,32 @@ def update(self, time_index: int) -> None:
             community.mortality_community()
             community.increase_age_community(timedelta64(1, "D"))
 
+        # Now that communities have been updated information required to update the
+        # litter model can be extracted
+        additions_to_litter = self.calculate_litter_additions()
+
+        # Update the litter pools
+        self.data.add_from_dict(additions_to_litter)
+
     def cleanup(self) -> None:
         """Placeholder function for animal model cleanup."""
+
+    def calculate_litter_additions(self) -> dict[str, DataArray]:
+        """Calculate the how much animal matter should be transferred to the litter."""
+
+        # Find the size of all excrement pools
+        remaining_excrement = [
+            community.excrement_pool.stored_carbon(self.data.grid.cell_area)
+            for community in self.communities.values()
+        ]
+
+        # After an update all excrement that isn't consumed is assumed to enter the
+        # litter, so stored energy of each pool is reset to zero
+        for community in self.communities.values():
+            community.excrement_pool.stored_energy = 0.0
```

Comment:
If I'm understanding correctly, this ordering will create a situation where the excrement pool is always empty when foraging begins. Right now, excretion and foraging and intertwined processes. Perhaps this is a sign they should be untangled. In the meantime, if we could use a slightly different assumption, like 50% loss per tick or something, I think that would work better.

Reply:
Yeah you are correct. I basically did this setting to zero as a first pass as I was sure how best to handle competition between excrement scavengers and microbial decomposition. If a percentage loss per tick would work for you then happy to go with that